### ISSUE TITLE
proper base selection when features are toggled via flags

### ISSUE THREAD

**Issue by ashmrtn** [alcionai/corso#2333]

**Comment by ashmrtn** [comment]
somewhat related to #2261

**Comment by meain** [comment]
@ashmrtn Do we have to consider this for incrementals and keep it associated to https://github.com/alcionai/corso/issues/2117 ? Permissions should be always available once we release it.

That said, we can keep this open if you want to use this issue to track using failed backups as base for future backups.

**Comment by meain** [comment]
>  ... we provide something like a `--no-permissions` flag. I don't know if we want that flag ...

I don't think we will be keeping around a flag like that. @gmatev , feel free to correct me if I am wrong.
